clients/mobile/src/database/index.ts

```diff
@@ -0,0 +1,17 @@
+import { DB } from 'react-native-sqlite-manager'
+
+import Migration from './Migration'
```

Comment:
since you setup `migrations` folder , so I think we should use something lie 
```
import SQLite from 'react-native-sqlite-storage';
import { MigrationManager } from 'react-native-sqlite-migration-manager';
import migration1 from './migrations/1';

const db = SQLite.openDatabase({ name: 'test.db', location: 'default' });

const migrations = [migration1];

const migrationManager = new MigrationManager(db, migrations);

migrationManager
  .migrate()
  .then(() => {
    console.log('Migrations applied successfully');
  })
  .catch((error) => {
    console.error('Error applying migrations:', error);
  });
```